addons/mail/static/src/components/chatter/chatter.xml

```diff
@@ -11,8 +11,8 @@
                     />
                     <t t-if="chatter.composerView">
                         <Composer
-                            className="'o_Chatter_composer'"
-                            classNameObj="{ 'o-bordered': chatter.hasExternalBorder }"
+                            class="o_Chatter_composer"
+                            t-att-class="{ 'o-bordered': chatter.hasExternalBorder }"
```

Comment:
Hello, I don't understand this change. This is a revert of what was done https://github.com/odoo/odoo/pull/80060/

If there is a bug with `className`/`classNameObj` is should be fixed instead?

Reply:
I am still not able to get the reason,
but after resizing window it goes for updating the props in `chatter_container` and then from there it rerenders the component. In the rerendering process it do rerender the composer properly, but when it is patched to the DOM it doesn't add the classes from `ClassNameObj`. And so these classes goes missing and the issue is being generated.

Reply:
Ok, I guess if the issue is resolved in 15.3 (owl 2) we can temporarily revert these lines in 15.1 and 15.2
cc @alexkuhn 

However you might need to rebase to fix ci/upgrade_enterprise?

Reply:
Ok..

Reply:
@seb-odoo Done!